### ISSUE TITLE
Figuring out the layout of the wiki

### ISSUE THREAD

**Issue by andresrcc** [cps-org/cps-wiki#1]
Thinking about the layout, we could use the [divio documentation system](https://documentation.divio.com/) and divide the wiki into 4 sections: Tutorials, How to Guides, Explanation and References.

**Comment by bretbrownjr** [comment]
I agree, and if you look at [the raw source of the existing Welcome page](https://github.com/cps-org/cps-wiki/wiki/Welcome/_edit), you'll see it is laid out that way.

It now occurs to me that [the Home page](https://github.com/cps-org/cps-wiki/wiki) is probably the best place for a Welcome page. @andresrcc, in the spirit of getting more people editing the wikis, can you merge those two things? I'm not sure if it's easier to migrate the contents of Welcome to Home or if there's a way to get GitHub to just point at Welcome by default.
